src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCost = 40;
+  let total = days * dailyRentalCost;
+  const threeDayDiscount = 20;
```

Comment:
It's better to use `const` instead of `let` for `threeDayDiscount` and `weeklyDiscount` since their values don't change. 🌟